From 371eb16f56b7b048955040a4d13b35f42141582d Mon Sep 17 00:00:00 2001
From: "Guoniu.Zhou" <guoniu.zhou@nxp.com>
Date: Thu, 17 Nov 2016 16:49:45 +0800
Subject: [PATCH 1277/5242] MLK-13478: Video: Correct the error macro
 definition in mxsfb.c

commit  1613a96bd114b369f312cc1b371687b91e040d50 from
https://source.codeaurora.org/external/imx/linux-imx.git

Correct a macro definition in mxsfb.c

In framebuffer driver, a macro define LCDIF_CTRL2n register bits[23-21] value
is 0x3, but according to reference manual, it should be 0x4, so correct it.

Signed-off-by: Guoniu.Zhou <guoniu.zhou@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 drivers/video/fbdev/mxsfb.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/video/fbdev/mxsfb.c b/drivers/video/fbdev/mxsfb.c
index 0c23932..73bcf14 100644
--- a/drivers/video/fbdev/mxsfb.c
+++ b/drivers/video/fbdev/mxsfb.c
@@ -128,7 +128,7 @@
 						 CTRL1_VSYNC_EDGE_IRQ)
 #define CTRL1_IRQ_STATUS_SHIFT			8
 
-#define CTRL2_OUTSTANDING_REQS__REQ_16		(3 << 21)
+#define CTRL2_OUTSTANDING_REQS__REQ_16		(4 << 21)
 
 #define TRANSFER_COUNT_SET_VCOUNT(x)	(((x) & 0xffff) << 16)
 #define TRANSFER_COUNT_GET_VCOUNT(x)	(((x) >> 16) & 0xffff)
-- 
1.7.9.5

